input/fsh/SD_Condition.fsh

```diff
@@ -24,7 +24,7 @@ Description: "Records the the primary cancer condition, the original or first tu
 * insert CancerConditionCommonRules
 * code from PrimaryOrUncertainBehaviorCancerDisorderVS (required)
 * code obeys primary-cancer-condition-code-invariant
-* stage.assessment only Reference(TNMStageGroup or TNMPrimaryTumorCategory or TNMRegionalNodesCategory or TNMDistantMetastasesCategory)
+* stage.assessment only Reference(CancerStageGroup or TNMPrimaryTumorCategory or TNMRegionalNodesCategory or TNMDistantMetastasesCategory)
```

Comment:
There is a potential problem here.  We give preference to the TNM Categories here, but elsewhere we leave the door open for other types in CancerStageGroup.hasMember (the slicing is #open). Here we close the door and allow ONLY TNM categories.
Possible solutions:
1) Eliminate this rule entirely -- allow anything
2) Limit to CancerStageGroup, remove the TNM categories -- because by design the CSG is the central object of a stage assessment.
3) Create a broad parent profile that embraces TNM **and** non-TNM Categories, and use it here instead of the TNM Categories.

I think I am partial to #2. The only reason the TNM categories are here is that I imagined a situation where the T, N, or M category resource could exist without the Cancer Stage Group, and so we'd want to reference them directly. Maybe that is not so important a scenario.